debian/mariadb-server-10.6.preinst

```diff
@@ -204,13 +218,15 @@ for dir in DATADIR LOGDIR; do
 done
 
 # creating mysql home directory
-if [ ! -d $mysql_datadir ] && [ ! -L $mysql_datadir ]; then
-	# Use mkdir option 'Z' to create with correct SELinux context.
+if [ ! -d $mysql_datadir ] && [ ! -L $mysql_datadir ]
+then
+  # Use mkdir option 'Z' to create with correct SELinux context.
   mkdir -Z $mysql_datadir
 fi
 
 # checking disc space
-if LC_ALL=C BLOCKSIZE= df --portability $mysql_datadir/. | tail -n 1 | awk '{ exit ($4>1000) }'; then
+if LC_ALL=C BLOCKSIZE='' df --portability $mysql_datadir/. | tail -n 1 | awk '{ exit ($4>1000) }'
```

Comment:
can we back port from #2549 regrarding this `df` to avoid merge conflicts.

Reply:
I'll do the backporting and settle PR

Reply:
Not there is #2612